### PR TITLE
PoC of AnsibleDriver [not for merge]

### PR DESCRIPTION
**Pull request by Settis**

### CODE REVIEW

File: kostyor/db/models.py [hunk 1]
```diff
@@ -66,6 +66,18 @@ class UpgradeTask(Base, KostyorModelMixin):
     upgrade_start_time = sa.Column(sa.DateTime)
     upgrade_end_time = sa.Column(sa.DateTime)
     status = sa.Column(sa.Enum(*constants.STATUSES))
+    driver_type = sa.Column(sa.Enum(*constants.DEPLOY_DRIVERS))
+    driver_config = sa.Column(sa.String(255))
```
Comment (sc68cal) [inline kostyor/db/models.py]: This will need a database migration. See #38

File: kostyor/core/driver.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import json
+
+
+class Driver(object):
```
Comment (sc68cal) [inline kostyor/core/driver.py]: I think that we should consider using #52 as the model - since steps may be internal to the implementation

Comment (ikalnitsky) [inline kostyor/core/driver.py]: Agree with Sean.

**Comment by sc68cal** [comment]
We'll pull this into a feature branch in the main repo.

**Pull request closed without merging**